<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileLibraryTab</class>
 <widget class="QWidget" name="ProfileLibraryTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Library</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="mainFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="mainHorizontalLayout">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QFrame" name="summaryFrame">
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QVBoxLayout" name="summaryLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="profileSummaryGroupBox">
              <property name="title">
               <string>Profile Summary</string>
              </property>
              <layout class="QVBoxLayout" name="profileSummaryLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QScrollArea" name="summaryScrollArea">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="summaryScrollAreaWidget">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>348</width>
                    <height>348</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="summaryScrollLayout">
                   <property name="spacing">
                    <number>2</number>
                   </property>
                   <property name="leftMargin">
                    <number>2</number>
                   </property>
                   <property name="topMargin">
                    <number>2</number>
                   </property>
                   <property name="rightMargin">
                    <number>2</number>
                   </property>
                   <property name="bottomMargin">
                    <number>2</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="profileSummaryLabel">
                     <property name="text">
                      <string>Select a profile from the library to view its summary.</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                     </property>
                     <property name="wordWrap">
                      <bool>true</bool>
                     </property>
                     <property name="margin">
                      <number>4</number>
                     </property>
                     <property name="openExternalLinks">
                      <bool>true</bool>
                     </property>
                     <property name="textInteractionFlags">
                      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="actionsGroupBox">
              <property name="title">
               <string>Actions</string>
              </property>
              <layout class="QVBoxLayout" name="actionsLayout">
               <item>
                <widget class="QPushButton" name="deleteProfileButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string>Delete the selected profile from disk</string>
                 </property>
                 <property name="text">
                  <string>Delete Selected Profile</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="actionsVerticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="libraryFrame">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QVBoxLayout" name="libraryLayout">
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="libraryLabel">
              <property name="styleSheet">
               <string>QLabel { font-weight: bold; font-size: 14px; padding: 6px; }</string>
              </property>
              <property name="text">
               <string>Profile Library</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="filterFrame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <layout class="QHBoxLayout" name="filterLayout">
               <property name="leftMargin">
                <number>6</number>
               </property>
               <property name="topMargin">
                <number>6</number>
               </property>
               <property name="rightMargin">
                <number>6</number>
               </property>
               <property name="bottomMargin">
                <number>6</number>
               </property>
               <item>
                <widget class="QLabel" name="filterLabel">
                 <property name="text">
                  <string>Filter:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="filterLineEdit">
                 <property name="placeholderText">
                  <string>Filter profiles by camera, make, model, ISO...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="refreshButton">
                 <property name="toolTip">
                  <string>Refresh profile list</string>
                 </property>
                 <property name="text">
                  <string>Refresh</string>
                 </property>
                 <property name="icon">
                  <iconset theme="view-refresh">
                   <normaloff>.</normaloff>.</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="profileTableWidget">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <property name="columnCount">
               <number>6</number>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Camera ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Make</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Model</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Shutter Speed</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ISO</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Date Created</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
